test_conformance/basic/test_vloadstore.cpp

```diff
@@ -360,53 +378,30 @@ int test_vload_local(cl_device_id device, cl_context context, cl_command_queue q
     cl_ulong localSize;
     int error = clGetDeviceInfo( device, CL_DEVICE_LOCAL_MEM_SIZE, sizeof( localSize ), &localSize, NULL );
     test_error( error, "Unable to get max size of local memory buffer" );
-    if( localSize > 10240 )
-        localSize = 10240;
-    if (localSize > 4096)
-        localSize -= 2048;
-    else
-        localSize /= 2;
+    localSize = std::min(localSize, (cl_ulong)2048);
```

Comment:
Think this is a change to the max local memory tested, for [CL_DEVICE_LOCAL_MEM_SIZE](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_DEVICE_LOCAL_MEM_SIZE) devices will probably report at least. Which would have been reduced to 8KB before, but now will be 2KB.

Were you trying to simplify the code with this chane, or was there a test performance reason behind it?

Same comment for the identical change in vstore.

Reply:
indeed I think I wanted to simplify those blurry conditions. Do you want me to restore original code ?

Reply:
@EwanC previous sizes reverted